### DISCUSSION TITLE
Acteur Détail de Reflet pour une ressource Consigne analogique (Apparition d'un curseur)

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#619] [Q&A]
Bonjour,

Coté programmation, j'ai une ressource consigne analogique définissant une tempo pour une alarme avec un type de reflet "Reflet analogique" qui est commandable. 
Coté synapps, j'ai créé un acteur "Détail de reflet". Tout fonctionne je peux écrire la valeur que je souhaite. Par contre, dès que je mets des valeurs limites (minimum et/ou maximum) sur la ressource, j'ai un curseur qui apparait sur ma scène lors d'une nouvelle session d'utilisateur.

Est-ce normal ?
Si Oui, pourrait-on avoir une case dans l'acteur pour dire que nous ne voulons pas ce curseur.

![1](https://github.com/witsa/synapps/assets/50445069/a62c277d-cbd0-4346-b516-b50922ffb2f5)

**Comment by manufitoussiwit** [comment]
Salut, tu trouveras une réponse ici : #524
